### DISCUSSION TITLE
Syntax for a negative controlled not

### DISCUSSION THREAD

**Discussion by BoltzmannEntropy** [QuantumBFS/Yao.jl#441] [Q&A] [answered]
Hello, 
What is the correct syntax in Yao for an empty circle? 
<img width="94" alt="image" src="https://user-images.githubusercontent.com/91342039/201486875-0a95da66-c1d8-4d03-95f8-4f73ff0ae2eb.png">

I of course know I can implement it with two X gates. 
Thanks,

**Comment by GiggleLiu** [comment, marked as answer]
You can use negative addresses to denote the inverse control qubits. In your plot, it is `control(6, (-1, 2), 6=>X)`.
